src/test/scala/cromwell/WdlThreeStepFixture.scala

```diff
@@ -0,0 +1,49 @@
+package cromwell
+
+import cromwell.binding.WdlBinding
+
+trait WdlThreeStepFixture {
```

Comment:
Ah okay.  Though I think you may be able to get rid of the `fixture` thing, that's kind of weird with only the single `binding` field.